tests/dummy/app/routes/nodes/detail/draft-registrations/detail.js

```diff
@@ -1,72 +1,14 @@
 import Ember from 'ember';
-import permissions from 'ember-osf/const/permissions';
 
 export default Ember.Route.extend({
     model(params) {
         let node = this.modelFor('nodes.detail');
-        let draft = this.store.peekRecord('draft-registration', params.draft_registration_id);
-        return Ember.RSVP.hash({
-            node: node,
-            draft: draft
-        });
+        this.store.adapterFor('draft-registration').set('namespace', 'v2/nodes/' + node.id);
```

Comment:
If the `DraftRegistrationAdapter` needs some custom behavior as a subclass of the `ApplicationAdapter`, then that logic should be implemented there. Is there a relationship on the Node model we can use for this?